docs/wiki/Solution creation.md

```diff
@@ -692,3 +718,74 @@ jobs:
 ```
 
 </details>
+
+# General solution creation
+
+When creating a solution that leverages CARML modules, there are several aspects to consider. This sub-section intends to provide you with a rough step-by-step guide to get you started.
+
+1. Identify the resources & deployment scope needed for your architecture
+
+    When you want to create your solution you should first gain an understanding of its designated architecture and and a results its inherently required services. For this and the subsequent steps, let's consider the following scenario:
```

Comment:
I think there's a typo but not sure how to correct it

Reply:
fixed